src/core_plugins/kibana/public/dashboard/dashboard.html

```diff
@@ -55,14 +57,20 @@
   <!-- Filters. -->
   <filter-bar state="state"></filter-bar>
 
-  <div ng-show="showEditHelpText()" class="text-center start-screen">
+  <div ng-show="getShowEditHelpText()" class="text-center start-screen">
     <h2>This dashboard is empty. Let's fill it up!</h2>
     <p>Click the <a class="btn btn-xs navbtn-inverse" ng-click="kbnTopNav.open('add'); toggleAddVisualization = !toggleAddVisualization" aria-label="Add visualization">Add</a> button in the menu bar above to add a visualization to the dashboard. <br/>If you haven't setup a visualization yet visit <a href="#/visualize" title="Visualize">"Visualize"</a> to create your first visualization.</p>
   </div>
 
+  <div ng-show="getShowViewHelpText()" class="text-center start-screen">
```

Comment:
TBH, when I read this name, it sounds like it could be returning a string ("help text").

> Also don't like is, because it makes it sound like it's actually checking if the help text is visible, but it's not initially.

What do you mean by "it's not initially visible"?

The way I think of it is that it's derived state, used to declaratively determine the state of the UI. "If the edit help text is visible, render this part of the template."